content/en/docs/examples/microservices-istio/add-istio/index.md

```diff
@@ -94,7 +94,10 @@ disrupt your application, it continues to run and serve user requests.
 
 1.  Check the Istio dashboard, using the custom URL you set in your `/etc/hosts` file
     [previously](/docs/examples/microservices-istio/bookinfo-kubernetes/#update-your-etc-hosts-configuration-file)):
-    [http://my-istio-dashboard.io/dashboard/db/istio-mesh-dashboard](http://my-istio-dashboard.io/dashboard/db/istio-mesh-dashboard).
+
```

Comment:
I thought the old way worked fine. At least when I clicked on the link it went right to it in my browser and no need to cut and paste.

Reply:
Same comment in other places.